src/Microsoft.Identity.Client/Platforms/iOS/AuthenticationContinuationHelper.cs

```diff
@@ -45,5 +45,26 @@ public static bool SetAuthenticationContinuationEventArgs(NSUrl url)
         {
             return WebviewBase.ContinueAuthentication(url.AbsoluteString);
         }
+
+        /// <summary>
+        /// Returns if the response is from the broker app
+        /// </summary>
+        /// <param name="sourceApplication">application bundle id of the broker</param>
+        /// <returns>True if the response is from broker, False otherwise.</returns>
+        public static bool IsBrokerResponse(string sourceApplication)
+        {
+            System.Diagnostics.Debug.WriteLine("IsBrokerResponse Called with sourceApplication {0}", sourceApplication);
```

Comment:
I can remove this before merge, but found them helpful when figuring the callback